woocommerce/woocommerce-android
# [WooExpress Removal] Remove site upgrade options

Part of: #11115

⚠️ Please don't merge yet.

### Description
This PR handles the second part of #11115, which is removing the ability to upgrade sites from the app:
1. Renames the "Upgrades" entry in the more menu screen to "Subscriptions"
2. Hides the "Subscriptions" entry for free trial sites.
3. Remove the "Upgrade now" button from the free trial banner and from the "Subscriptions" screen.

**Same as the previous PR, this doesn't include any code cleanup, it just hides the different items, code cleanup will be handled separately**

### Testing instructions
1. Sign in using WordPress.com.
2. Choose a WooExpress website.
3. Confirm that the "Upgrades" button was renamed to "Subscriptions"
5. Click on it and confirm the functionality still works.
6. Choose a free trial website (create one on the web if needed)
7. Confirm the "Upgrade Now" button of the free trial website has been removed.
8. Confirm the "Subscriptions" entry in the more menu is not shown.
9. Confirm the Onboarding tasks don't include neither "Launch your Store" nor "Name your Store".

### Images/gif
